pkg/detectors/wiz/wiz.go

```diff
@@ -69,14 +69,6 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 				s1.SetVerificationError(verificationErr, idMatch, secretMatch)
 			}
 
-			// This function will check false positives for common test words, but also it will make sure the key appears 'random' enough to be a real key.
-			if !s1.Verified {
-				if detectors.IsKnownFalsePositive(idMatch, detectors.DefaultFalsePositives, true) ||
-					detectors.IsKnownFalsePositive(secretMatch, detectors.DefaultFalsePositives, true) {
```

Comment:
This custom logic wasn't moved into a `CustomFalsePositiveChecker` implementation because `secretMatch` is no longer available by the time that runs. This type of change (which "narrows" false positive detection) was made for multiple other detectors in #2643 so I'm not worried about it.